extensions/blocks/podcast-player/templates/podcast-title.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
```

Comment:
I wish there was a better way to do this? 

It looks like we can set certain variables to be ignored from this `UNdefinedVariable` check.

https://github.com/sirbrillig/phpcs-variable-analysis#customization

I proposed we 

* update the var from `$data` to `$templateProps` 
* add a rule to the `.phpcs.xml.dist` under the `validUndefinedVariableNames` property to ignore the var `$templateProps`.
* update the code in the `render` so that it doesn't perform an `extract()` and simply passes the `$templateProps` array. 
* the template can extract the data (or not) as required without having to worry about disabling `phpcs` warnings.

@jeherve what do you think about this? Having to litter each template with these disable rules feels really horrible.

Reply:
Yes, that sounds like a good plan!